doc/nrf/app_dev/device_guides/nrf54l/vpr_flpr.rst

```diff
@@ -0,0 +1,35 @@
+.. _vpr_flpr_nrf54l:
+
+Working with the FLPR core
+##########################
+
+The nRF54L15 SoC has a dedicated RISC-V CPU (VPR), a fast lightweight peripheral processor (FLPR) optimized for software-defined peripherals (SDP).
```

Comment:
This sentence can be interpreted as there were two CPUs: VPR for general usage and FLPR for SDPs.

```suggestion
The nRF54L15 SoC has a dedicated VPR CPU (RISC-V architecture), named "fast lightweight peripheral processor" (FLPR) optimized for software-defined peripherals (SDP).
```